android/assets/jsons/Civ V - Gods & Kings/UnitPromotions.json

```diff
@@ -358,29 +358,26 @@
 		"uniques": ["[+34]% Damage when intercepting"],
 		"unitTypes": ["Fighter"]
 	},
-	/*
 	{
 		"name": "Dogfighting I",
-		"uniques": ["Bonus when performing air sweep [33]%"], // todo
+		"uniques": ["[+33]% Strength bonus when performing air sweep"],
```

Comment:
We generally try not to use the words 'bonus' in uniques, as "-33% Strengh bonus" sounds a bit weird. I'd write this as "[+33]% Strength when performing an air sweep"

Reply:
done